test/Spec.hs

```diff
@@ -80,7 +81,7 @@ tests docker0 = testGroup "Tests"
           (expectFailure (ioProperty (prop_echoParallelOK True <$> mkEnv)))
       ]
   , testGroup "ProcessRegistry"
-      [ testProperty "sequential" (prop_processRegistry markovGood)
+      [ QCSM.testProperty markovGood "sequential" prop_processRegistry
```

Comment:
I think we should flip the arguments around here...

```haskell
QCSM.testProperty "sequential" prop_processRegistry markovGood
```
Is easier to read in my opinion.